Automattic/wp-calypso
# Untangling: Add Settings -> Caches subtab

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9616

## Proposed Changes

This PR adds the Settings -> Caches under the `untangling/hosting-menu` flag.

<img width="599" alt="image" src="https://github.com/user-attachments/assets/dc0969db-b516-4db7-9acc-e8cea719b022">


The setting comes from Server Settings -> Performance Optimization card.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Settings -> Caches subtab.
    - Verify that this setting is working.
5. Click a Simple site from the site list
    - Verify that this setting is not available.
7. Regression testing:
    - Go to Server Settings
    - Verify that the original Performance Optimization card is still working.

## Pre-merge Checklist
